### PR TITLE
{bio}[foss/2021b] Trycycler v0.5.3 w/ Python 3.9.6

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

Version update and also adding missing dependencies as per #15864 
See https://github.com/rrwick/Trycycler/wiki/Software-requirements

Depends on:
* [x] #15858

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 4 out of 4 (1 easyconfigs in total)
node029.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/ae55c0847446ab3810f15fccfb7f92ef for a full test report.

**Comment by jfgrimm** [comment]
@boegelbot please test @ generoso

**Review by jfgrimm — APPROVED** [review]
LGTM

File: easybuild/easyconfigs/t/Trycycler/Trycycler-0.5.3-foss-2021b.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+sources = ['v%(version)s.tar.gz']
+checksums = ['9a3a512284a91c8a221e3f4dedbc8acabd8f7b93937ad46402d93eee6c7ee95b']
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('R', '4.1.2'),
+    ('edlib', '1.3.9'),
+    ('miniasm', '0.3-20191007'),
+    ('minimap2', '2.24'),
```
Comment (jfgrimm) [inline easybuild/easyconfigs/t/Trycycler/Trycycler-0.5.3-foss-2021b.eb:21]: drop to 2.22 to match other easyconfigs in this toolchain generation

```suggestion
    ('minimap2', '2.22'),
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/t/Trycycler/Trycycler-0.5.3-foss-2021b.eb (hunk 1 above)
Comment (lexming) [reply easybuild/easyconfigs/t/Trycycler/Trycycler-0.5.3-foss-2021b.eb]: thanks for the fix!

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node029.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/315eb43efdf9d7367fdcace8b4373b5a for a full test report.

**Review by jfgrimm — APPROVED** [review]

**Pull request merged**